src/org/usfirst/frc2175/Robot.java

```diff
@@ -22,9 +23,10 @@
             new DriverStation(robotConfig, new DeadbandCalculator());
     private final RobotSubsystems robotSubsystems =
             new RobotSubsystems(robotConfig, driverStation);
+    private final IntakeConfig intakeConfig = new IntakeConfig();
     private final JoystickEventMapper joystickEventMapper =
-            new JoystickEventMapper(robotConfig, driverStation,
-                    robotSubsystems);
+            new JoystickEventMapper(robotConfig, driverStation, robotSubsystems,
+                    intakeConfig);
```

Comment:
`intakeConfig` should be included in `robotConfig`, so this shouldn't be necessary.